content/docs/next-release/deployment.md

```diff
@@ -306,6 +306,25 @@ Rollover index management strategy is more complex than using the default daily
 To learn more about rollover index management in Jaeger refer to this
 [article](https://medium.com/jaegertracing/using-elasticsearch-rollover-to-manage-indices-8b3d0c77915d).
 
+#### Elasticsearch ILM support
+{{< warning >}}
+Experimental Feature, Added in release 1.22.
+
+Supported versions: 7.x
+{{< /warning >}}
+[Elasticsearch ILM](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html) helps us automatically manage indices according to our performance, resiliency, and retention requirements.
+For example, you could use ILM to:
+
+* Spin up a new index when an index reaches a certain size or number of documents
+* Create a new index each day, week, or month and archive previous ones
+* Delete stale indices to enforce data retention standards
+
```

Comment:
I would aim to make the steps below as simple as possible for users to follow. A suggestion is to provide code that they can copy-paste where practical, even if it's just an example that they can modify later on, and avoid indirection. Having all the steps in one place just for ILM means we can avoid touching any existing manual rollover documentation.

I've provided a suggestion below.